### ISSUE TITLE
missing field to enter the password in the settings

### ISSUE THREAD

**Issue by WiliTest** [ispysoftware/iSpy#41]
Hi!
I want to set a password to lock the app so people can't delete the video. But there is no password field (as mentioned in the [docs ](https://www.ispyconnect.com/userguide-settings.aspx)

This picture explains the problem 

https://i.imgur.com/OgPzBPR.png

**Comment by ispysoftware** [comment]
It's been updated to support multiple users - click on the permissions
button

**Comment by WiliTest** [comment]
thanks for your reply, I tried the new version but nothing changes (I installed it on my main win10 admin user account and ran it from there).

**Comment by ispysoftware** [comment]
Click in Permissions in settings. Set a password there or click Add group
to create a new subset of permissions
[image: Inline image 1]

**Comment by WiliTest** [comment]
Oh yes, I apologized, the password field is on the top of the permission popup windows. It works, thanks!


ps: I don't knwo if it's a bug or if it's normal but when I sign in to another user (user2) and let ispy opened in the user1, launching ispy in user2 won't work (anything start). I must first sign in again in user1, close ispy and come back in user2.

Thanks a lot for your awesome work!

**Comment by Elliott71** [comment]
I need help! I clicked to have it password protected and I did not get to set a password! No pop ups no nothing. Now I am locked out of the app! I clicked to have it run minimized on start up as well. I have uninstalled the app and even tried to do a recovery, nothing works. I even tried downloading the the ispyconnect and still nothing! I love the app and wanted to purchase the full version. Is there anything I can do??
